system/Filters/Filters.php

```diff
@@ -474,6 +477,36 @@ protected function processFilters(string $uri = null)
 		}
 	}
 
+	/**
+	 * filter alias to class
+	 *
+	 * @return type
```

Comment:
docblock can be updated to:

```suggestion
       * @return void
       * @throws FilterException
```